variables.tf

```diff
@@ -23,6 +23,24 @@ variable "create_cos_instance" {
   default     = true
 }
 
+variable "create_hmac_key" {
+  description = "Set as true to create a new HMAC key for the Cloud Object Storage instance."
+  type        = bool
+  default     = true
+}
+
+variable "hmac_key_name" {
+  description = "Set as true to create a new HMAC key for the Cloud Object Storage instance."
```

Comment:
incorect description

Reply:
marking as unresolved, not addressed yet

Reply:
updated, was just marking off as i did them locally to not get confused. will just leave unresolved next time.